apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -120,6 +138,124 @@ describe('/research-outputs/ route', () => {
         researchOutputId,
       );
     });
+    test('Should return 404 when you are trying to get a draft and are not part of the team', async () => {
+      userMockFactory.mockReturnValueOnce(createUserResponse());
+
+      const researchOutputResponse = {
+        ...getResearchOutputResponse(),
+        published: false,
+        teams: [{ id: 'nop', displayName: 'team 1' }],
+      };
+
+      researchOutputControllerMock.fetchById.mockResolvedValueOnce(
+        researchOutputResponse,
+      );
+
+      const response = await supertest(app).get('/research-outputs/123');
+
+      expect(response.status).toEqual(404);
+    });
+    test('Should return 404 when you are trying to get a draft and are not part of the working group', async () => {
+      userMockFactory.mockReturnValueOnce(createUserResponse());
```

Comment:
same as above, in fact make sure you defined both teams and working groups for the user, that way we know we didn't accidentally pass it because something else matched